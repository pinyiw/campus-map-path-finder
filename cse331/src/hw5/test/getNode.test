# Create a graph
CreateGraph g

# Add nodes
AddNode g a
AddNode g b
AddNode g c

# Print all the nodes
ListNodes g