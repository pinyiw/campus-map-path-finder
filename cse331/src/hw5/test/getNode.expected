# Create a graph
created graph g

# Add nodes
added node a to g
added node b to g
added node c to g

# Print all the nodes
g contains: a b c